core/mondoo-github-incident-response.mql.yaml

```
@@ -49,7 +49,7 @@ packs:
         title: GitHub Organization MFA status
         docs:
           desc: |
-            This query retrieves whether MFA is required for users of the organization.
+            This query retrieves whether MFA is required for users of the organization. If this value is null the API token used to query the information did not have owner permissions on the organization.
```

```suggestion
            This query retrieves whether multi-factor authentication is required for users of the organization. A null value means the API token used to query the information doesn't have sufficient permissions in the organization. The API token must have owner permissions in the organization to access this data.
```
It's wordy but I'm worried that if we don't spell it out at length the reader won't get it.